src/components/MainSidebar/DimensionsList/DimensionsList.module.css

```diff
@@ -1,4 +1,5 @@
 .scrollbox {
+    height: calc(100% - 57px);
```

Comment:
This is not ideal since it depends on the height of a different component (the search box). But I don't know of a better way to do this.

Reply:
Replace this by adding `display: flex; flex-direction: column;` to the parent?
![image](https://github.com/dhis2/line-listing-app/assets/12590483/b60af193-9e51-451e-aa24-16990c2f11b7)

Reply:
Awesome, works well.